lessons/Data-Science/ai-decision-trees/langs/en-us/index.adoc

```diff
@@ -0,0 +1,278 @@
+
+...but how do we make such decisions?!
+
+It turns out, there's an algorithm for that, and it's relatively straightforward.
+
+
+=== Investigate
+
+Have you ever done some online shopping—say, for a new pair of sneakers—only to discover that, for the next several days, you encounter _advertisements for sneakers_ lurking in every corner of the internet that you visit?!
+
+Is it a coincidence? No. Computer cookies are small data files stored locally on your device. One particular kind of cookie, the tracking cookie, allows AI designed for marketing to use your individual browsing habits to decide which ads you will be the most susceptible to.
+
+We're going to create a decision tree that predicts whether or not different customers at a particular online store will purchase a video game or not. To do so, we must first train the computer! We will use a training dataset that characterizes 14 different shoppers and then indicates whether each one purchased a video game or not.
+
+@QandA{
+@Q{With your partner, look over the @handout{decision-tree-data.adoc, Training Dataset}. What do you Notice? What do you wonder?}
+@A{Possible responses:}
+@A{Individuals in their twenties always buy the video game.} 
+@A{There are only three new customers; two out of three times, new customers buy the video game.}
+@Q{Can you foresee any problems with making a decision tree based on this dataset? If so, what are they?}
+@A{Responses will vary.}
+}
+
+One problem with this dataset is that _age is continuous_. That won’t work! We need to break these ages down into different groups - which will become different _branches_ that grow out of a _decision node_. *For now, let’s agree to create three groups: teenagers; twenties; and thirties.*
+
+@lesson-instruction{
+- We will complete @printable-exercise{level-1.adoc} together, starting with "age" as the root node.
+- As we create the first level of our tree, you will discover a tool that we use to complete all the levels of our model: the decision stump.
+}
+
+As we move down the tree, our job is to figure out _which questions_ we should ask and _when_ we should ask them... just like when we play 20 Questions! Decision stumps will help us decide which questions produce a greater information gain.
+
+@strategy{Why Start the Tree with "Age"?}{
+Students will likely notice that we seemingly arbitrarily started the tree with "age" as the root node. _Extremely perceptive_ students may notice that for both "age" and "interest", the likelihood of a correct prediction is 10/14. In other words, starting with "interest in game" produces the same information gain as starting with "age" as the root... *so how do we decide?*
+
+It turns out there is no *one* correct way to build a decision tree. In general, however, we want to avoid tall, skinny trees that pose one useless question after the other. Rather, it is beneficial to start with an attribute that will result in a _wider_ tree.
```

Comment:
Confusing wording. I think it would be clearer if you wrote "It turns out there's more than one correct way to build a decision tree!"